### ISSUE TITLE
Connection Problems

### ISSUE THREAD

**Issue by ghost** [nicehash/NiceHashMiner-Archived#1124]
I keep getting this message.
A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
|stratum  Reconnecting in 3 seconds...

**Comment by wfoster** [comment]
OfficerButt

What is your setup? MoBo, GPU and how many, OS, PSU

Thanks

**Comment by ghost** [comment]
GA-970A-DS3P, AMD FX3650 and nvidia gtx 1060 6gb, windows 10, EVGA 650W G2 8gb ram 1600mhz.

**Comment by wfoster** [comment]
Have you tried using location closest to you?
Europe, USA, China, Japan, India, Brazil

**Comment by ghost** [comment]
Yes.. ive even tried using everyone but it still says Connection lost...

**Comment by papampi** [comment]
try it with a vpn 
private internet access solved my probem
seems like some ips are blocked by nicehash

also if your router has vulnerability protection, turn  it off
